src/content/en/updates/2020/03/devtools.md

```diff
@@ -0,0 +1,83 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2020-03-09 #}
+{# wf_published_on: 2020-03-09 #}
+{# wf_tags: chrome81, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 82) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Emulate vision deficiencies 
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e847a020093fbd3c367eda02a7008feae5e28641 #}
+
+Send feedback to [Chromium issue #1003700](https://crbug.com/1003700).
+
+## Clear filters button in Network panel {: #TODO }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/2604c1917006056e82007ef892e6223218cbc346 #}
+
+Send feedback to [Chromium issue #1056666](https://crbug.com/1056666).
+
+## Dock-to-left in the Command Menu {: #dock-to-left }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/7ff2e8eda31b86a8e249983f182e20e65c303556 #}
+
+Send feedback to [Chromium issue #1011679](https://crbug.com/1011679).
+
+## COEP {: #COEP }
+
+{# Not sure if COEP is launching behind a flag in Chrome 82. We usually don't cover web platform or DevTools experiments. #}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/a2afe0b0d0b307a2bc2c1a8e9ed5c97f1b89569c #}
+
+Send feedback to [Chromium issue #1051466](https://crbug.com/1051466).
+
+## Lighthouse 5.7.1 {: #lighthouse }
+
+New audits:
+
+* [CLS](https://github.com/GoogleChrome/lighthouse/commit/a8a0f8fd369faaa23b97d8803bfc220db31e4769#diff-eb7e207cabf22c8000b5abe8f18c6daf)
+* [Legacy JS](https://github.com/GoogleChrome/lighthouse/pull/10303)
+* [Maskable icon](https://github.com/GoogleChrome/lighthouse/pull/10370)
```

Comment:
Should we update the earlier DevTools release notes with maskable icon support to reference the newly added Lighthouse support?

Actually, I just realized that the maskable icon feature in DevTools wasn’t included in [the M81 release notes](https://developers.google.com/web/updates/2020/01/devtools). @kaycebasques could you please add this? We already have feature documentation [here](https://web.dev/maskable-icon/) so you could probably re-use some text.

Reply:
I can add it to these release notes instead and say "we missed this in m81". It will get more awareness that way.

Reply:
I actually can't even see any of these new audits: https://github.com/google/WebFundamentals/pull/8548#issuecomment-597393736

I can mention the maskable audit feature from m81 in these release notes and just say "we missed this in m81 but are covering it now"... that way it gets more awareness. There will be a few people who say "why are you covering something from m81 in m82?" but the increased overall awareness is worth making a few people annoyed/confused IMO.

Reply:
maskable icons section added